huntsman/scheduler/observation.py

```diff
@@ -79,3 +81,35 @@ def add_field(self, new_field, new_exptime):
 
     def __str__(self):
         return "DitheredObservation: {}: {}".format(self._field, self._exptime)
+
+
+class DitheredFlatObservation(DitheredObservation):
+    """ A DitheredObservation specifically for flat fields."""
+
+    def __init__(self, position, pattern=dither.dice9, n_positions=9,
+                 dither_offset=0.5*u.arcmin, random_offset=0.5*u.arcmin,
+                 *args, **kwargs):
+        """
+        Args:
+            position (str): Center of field, can be anything accepted by
+                `~astropy.coordinates.SkyCoord`.
+        """
+        # Create the observation
+        field = Field('Flat Field', position.to_string('hmsdms'))
+        super().__init__(field=field, *args, **kwargs)
+
+        # Listify the exposure time
+        self.exptime = [self.exptime for _ in range(n_positions)]
+
+        # Setup the dither fields
+        dither_coords = dither.get_dither_positions(
+                field.coord, n_positions=n_positions, pattern=pattern,
+                pattern_offset=dither_offset, random_offset=random_offset)
```

Comment:
I'm confused as to why this is not part of `DitheredObservation` already.. :confused:

Reply:
It certainly seems like it should be.

Reply:
It must be doing it somehow already, because observing with `DitheredObservation` has actually dithered in the past.

Reply:
I thought there was some conversation around the fact that this should be removed from here.